### ISSUE TITLE
Detected that custom integration 'eskom_loadshedding' calls `async_add_job`, which is deprecated

### ISSUE THREAD

**Issue by SunilKhusal** [swartjean/ha-eskom-loadshedding#41]
## Version of the custom_component

Latest Version v1.1.2

## Configuration

```yaml

Detected that custom integration 'eskom_loadshedding' calls `async_add_job`, which is deprecated and will be removed in Home Assistant 2025.4; Please review https://developers.home-assistant.io/blog/2024/03/13/deprecate_add_run_job for replacement options at custom_components/eskom_loadshedding/__init__.py, line 62: hass.async_add_job(, please create a bug report at https://github.com/swartjean/ha-eskom-loadshedding/issues

```

## Describe the bug
I am getting the above error on my log and thought of informing you of the deprication.

## Debug log

```text

Add your logs here.

```

**Comment by hawky358** [comment]
@SunilKhusal  I created a pull request for this issue. #43 

Hopefully @swartjean is still working on this then he can take a look to merge, otherwise you can incorporate the change into your HASS.

**Comment by swartjean** [comment]
This should be fixed now, see #43.
